tests/test_core_pools.py

```diff
@@ -0,0 +1,10 @@
+from gen_core_pools import is_core_pool
+
+
+def test_is_core_pool():
+    """
+    confirm wstETH-WETH is a core pool
+    """
+    assert is_core_pool(
+        "mainnet", "0x93d199263632a4ef4bb438f1feb99e57b4b5f0bd0000000000000000000005c2"
```

Comment:
For the next version, would be nice to put in a test for each of these odd types, but as discussed in other convos, that can be for a next PR/this is fine.